### PR TITLE
update remaining tests

### PR DESCRIPTION
**Pull request by morningvera**

### CODE REVIEW

File: rest/models/snapshot.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ type GetAllTickersSnapshotParams struct {
 // GetAllTickersSnapshotResponse is the response returned by the GetAllTickersSnapshot method.
 type GetAllTickersSnapshotResponse struct {
 	BaseResponse
-	Snapshots []TickerSnapshot `json:"ticker,omitempty"`
+	Tickers []TickerSnapshot `json:"tickers,omitempty"`
```
Comment (morningvera) [inline rest/models/snapshot.go:22]: generally i think we should make field names match the json tags

File: rest/quotes/quotes_test.go [hunk 2]
```diff
@@ -56,8 +57,8 @@ func TestListQuotes(t *testing.T) {
 	"request_id": "a47d1beb8c11b6ae897ab76cdbbf35a3",
 	"next_url": "https://api.polygon.io/v3/quotes/AAPL?cursor=YWN0aXZlPXRydWUmZGF0ZT0yMDIxLTA0LTI1JmxpbWl0PTEmb3JkZXI9YXNjJnBhZ2VfbWFya2VyPUElN0M5YWRjMjY0ZTgyM2E1ZjBiOGUyNDc5YmZiOGE1YmYwNDVkYzU0YjgwMDcyMWE2YmI1ZjBjMjQwMjU4MjFmNGZiJnNvcnQ9dGlja2Vy",
 	"results": [
-		` + quote1 + `,
-		` + quote2 + `
+` + indent(true, quote1, "\t\t") + `,
```
Comment (morningvera) [inline rest/quotes/quotes_test.go:60]: i realized we weren't indenting these correctly while updating the snapshot tests so i wrote a simple helper. it's just copied in the various test files for now

File: rest/models/snapshot.go [hunk 3]
```diff
@@ -73,125 +73,122 @@ type GetCryptoFullBookSnapshotResponse struct {
...
-	Day              DaySnapshot       `json:"day"`
-	LastQuote        LastQuoteSnapshot `json:"lastQuote"`
-	LastTrade        LastTradeSnapshot `json:"lastTrade"`
-	Minute           MinuteSnapshot    `json:"min"`
-	PrevDay          DaySnapshot       `json:"prevDay"`
-	Ticker           string            `json:"ticker"`
-	TodaysChange     float64           `json:"todaysChange"`
-	TodaysChangePerc float64           `json:"todaysChangePerc"`
-	Updated          int64             `json:"updated"`
+	Day              DaySnapshot       `json:"day,omitempty"`
```
Comment (mcdayoub) [inline rest/models/snapshot.go:76]: Looks like I `omitted` the `omitempty`s

File: rest/trades/trades_test.go [hunk 4]
```diff
@@ -125,35 +138,50 @@ func TestGetLastCryptoTrade(t *testing.T) {
...
 	})
 
 	assert.Nil(t, err)
-	assert.Equal(t, &expectedResponse, res)
+	b, err := json.MarshalIndent(res, "", "\t")
+	assert.Nil(t, err)
+	assert.Equal(t, expectedResponse, string(b))
+}
+
+func registerResponder(url string, body string) {
```
Comment (mcdayoub) [inline rest/trades/trades_test.go:168]: I like moving `registerResponder` into its own func. Nice.

**Review by mcdayoub — APPROVED** [review]

**Pull request merged**